sdccc/src/main/java/com/draeger/medical/sdccc/messages/MessageStorage.java

```diff
@@ -1426,9 +1426,11 @@ public GetterResult<MessageContent> getInboundMessagesByBodyTypeAndSequenceId(
                             messageContentRoot.get(MessageContent_.direction), CommunicationLog.Direction.INBOUND),
                     criteriaBuilder.exists(mdibVersionGroupSubQuery)));
 
-            messageContentQuery.orderBy(criteriaBuilder.asc(messageContentRoot
-                    .join(MessageContent_.mdibVersionGroups)
-                    .get(MdibVersionGroupEntity_.mdibVersion)));
+            messageContentQuery.orderBy(
+                    criteriaBuilder.asc(messageContentRoot
+                            .join(MessageContent_.mdibVersionGroups)
+                            .get(MdibVersionGroupEntity_.mdibVersion)),
+                    criteriaBuilder.asc(messageContentRoot.get(MessageContent_.timestamp)));
```

Comment:
Please add a unit test to TestMessageStorage for this.

Reply:
Unfortunately, in the case of TestMessageStorage. the reports in the DB output have exactly the same order in which they were put in even without the additional sorting clause. I can hence make sure that the clause is covered by a test case and that the resulting order is checked, but I cannot construct a Test case that fails when this clause is not present.
Would the former be sufficient?

Reply:
Not necessarily, you can add messages with custom timestamps and then check that the one with the lowest timestamp is output first, regardless of when it was added.

Reply:
Done (It is possible to manipulate the timestamps using Mock Messages).